.generator/src/generator/templates/api_client.j2

```diff
@@ -70,26 +63,11 @@ class ApiClient(object):
         self.close()
 
     def close(self):
-        if self._pool:
-            self._pool.close()
-            self._pool.join()
-            self._pool = None
-            if hasattr(atexit, "unregister"):
-                atexit.unregister(self.close)
+        pass
```

Comment:
do we need this method at all?

Reply:
I think it makes some sense to make it part of the interface? I need to look into closing the urllib pool manager as well.

Reply:
Done, I added the clear we had in the tests.